_source/logzio_collections/_log-sources/oke.md

```diff
@@ -0,0 +1,119 @@
+Oracle Cloud Infrastructure Container Engine for Kubernetes (OKE) is a fully-managed, scalable, and highly available service that you can use to deploy your containerized applications to the cloud.
+This implementation uses a Filebeat DaemonSet to collect Kubernetes logs from your OKE cluster and ship them to Logz.io.
+
+You have 3 options for deploying this Daemonset:
+
+* Standard configuration
+* Autodiscover configuration - the standard configuration which also uses Filebeat's autodiscover and hints system
+* Custom configuration - upload a Logz.io Daemonset with your own configuration
+
+
+#### Deploy Filebeat as a DaemonSet on Kubernetes
+
+<div class="tasklist">
+
+**Before you begin, you'll need**:
+
+* Destination port 5015 open on your firewall for outgoing traffic
+
+##### Store your Logz.io credentials
+
+Save your Logz.io shipping credentials as a Kubernetes secret.
+
+Customize the command to your specifics:
+
+* {% include log-shipping/replace-vars.html token=true %}
+* {% include log-shipping/replace-vars.html listener=true %}
+* Replace `<<CLUSTER-NAME>>` with your cluster's name. If you manage Kubernetes in AWS or Azure, you can find it in your admin console. Alternatively, you can run the following to obtain your cluster name: `kubectl cluster-info`
```

Comment:
please explain where we can find it in Oracle and not AWS / Azure :)

Reply:
My bad. Fixec it.